src/js/common/components/authentication/VerifyPrompt.jsx

```diff
@@ -36,4 +45,8 @@ class VerifyPrompt extends React.Component {
   }
 }
 
+VerifyPrompt.propTypes = {
+  verifyUrl: PropTypes.string
```

Comment:
Should this be required, and same for `LoginPrompt`'s `loginUrl`? (instead of requiring them in`RequiredLoginView`?)